src/term/net_to_term.rs

```diff
@@ -211,6 +214,42 @@ impl<'a> Reader<'a> {
 
     term
   }
+
+  /// Enters both ports 1 and 2 of a node  
+  /// If both ports are connected to the same node, and that node Kind is the same as the given node Kind,  
+  /// Reads the port 0 of the connected node, and returns that term,  
+  /// Otherwise, returns the terms on ports 1 and 2 of the given node.
```

Comment:
Maybe add that these two inets (before and after the transformation) always have equivalent outcomes, so this is why we're able to do it.